### ISSUE TITLE
Wont Register Config

### ISSUE THREAD

**Issue by XDragonSlayer81** [SlimeKnights/TinkersToolLeveling#91]
Have tried downloading different versions of forge and tinkers tool leveling itleast for 1.11.2 and 1.12.2 and i am not able to access the config option on the in game settings.

**Comment by bonii-xx** [comment]
Yes, we do not add the config in the ingame menu. You can only canfigure it via the config file.
